init.org

```diff
@@ -1246,7 +1246,8 @@
           ;; terminals which don't specify any special key combinations for the
           ;; reclaimed keys :cry:.
           ;; (define-key function-key-map [C-i]  "	")
-          ;; (define-key function-key-map [C-m]  "
")
+          ;; (define-key function-key-map [C-m]  "
+")
```

Comment:
This would introduce a syntax error since `")` isn't commented out

Reply:
Oh, oops! I must've clicked Return by accident!

My apologies!

Reply:
Ah, my bad, sorry. Classic github being weird. This line was fine the way it was. There was a literal "^M" character here which it looks like Github interprets as a newline. It'd be best if you could leave this the way it already is in origin/master. It needs to be ^M because we're translating from the custom key code [C-m] into the legacy key code ^M (same as the return key). Also for some reason github is being very slow for me atm so I'll look into any changes tomorrow.

S.N. No need to apologise. You're contributing here so I should be thanking you :-).

Reply:
Aha!

Nice catch! All solved now 6a34478a4271477891f259a79eea0efaa1810113.

Keep up the great work. I learnt quite a few things from reading this piece.